src/components/MoteresponsColumn.tsx

```diff
@@ -73,10 +85,13 @@ export default function MoteresponsColumn({ dialogmote }: Props): ReactElement {
     <ResponsColumn>
       <ResponseEntry deltaker={"Arbeidstaker"} respons={arbeidstakerRespons} />
       <ResponseEntry deltaker={"Arbeidsgiver"} respons={arbeidsgiverRespons} />
-      {behandlerSvar && (
+      {dialogmote.behandler && (
         <ResponseEntry
           deltaker={"Behandler"}
-          respons={{ harLest: false, svar: behandlerSvar }}
+          respons={{
+            harLest: false,
+            svar: behandlerSvar ? behandlerSvar : undefined,
```

Comment:
`behandlerSvar` ble brukt her for om man skulle vise noe, men `dialogmote.behandler` er riktigere.